recipe/meta.yaml

```diff
@@ -98,12 +98,12 @@ source:
       - optional-install.patch
 
 build:
-  number: 5
+  number: 6
   skip: true  # [cuda_compiler_version == "None"]
   skip: true  # [cuda_compiler_version == "10.2"]
   skip: true  # [cuda_compiler_version == "11.0"]
   skip: true  # [cuda_compiler_version == "11.1"]
-  skip: true  # [ppc64le]
+  skip: true  # [cuda_compiler_version == "11.2"]
```

Comment:
sorry why is this necessary? can we not be compatible with both using little effort?

Reply:
How much longer do you want me to build for 11.2? The 11.2 build takes to long, so I will need to reduce the number of target archs.

Reply:
can you install the newer cuda stuff with pytorch or tensorflow from 11.2? if so then that is ok with me.

We just don't have cuda != 11.2 for either of those.

Reply:
For clarity, the addition of ppc64le is unrelated to the removal of 11.2.

I think you would need an 11.2 build of magma to build pytorch with 11.2. I don't think you can use 11.8 build of magma to build pytorch with 11.2, so I'll try to get the 11.2 build passing again.

Reply:
we could go the other way too, and try to build pytorch and ensorflow with 11.8

Reply:
In term of migrating to CUDA 11.8 both PyTorch ( https://github.com/conda-forge/pytorch-cpu-feedstock/pull/195 ) and TensorFlow ( https://github.com/conda-forge/tensorflow-feedstock/pull/344 ) have updated

Regarding CUDA 11.2, PyTorch dropped it ( https://github.com/conda-forge/pytorch-cpu-feedstock/pull/195 ) and TensorFlow is planning to ( https://github.com/conda-forge/tensorflow-feedstock/pull/347#issuecomment-1805196832 )

So maybe we can simplify the builds here

Reply:
Thanks! I'll keep that in mind for the next release!

Reply:
Note: Tracking in issue ( https://github.com/conda-forge/libmagma-feedstock/issues/12 )